ftplugin/markdown.vim

```diff
@@ -0,0 +1,32 @@
+
+
```

Comment:
Blank lines to serve a purpose? Minor aesthetics, but still.

Reply:
Nope. Good catch!